README.md

```diff
@@ -246,6 +248,76 @@ expect(wrapper.find('#child')).to.have.descendants('#last')
 expect(wrapper).to.not.have.descendants('#root1')
 ```
 
+#### `descendantsCount(selector)`
+
+| render | mount | shallow |
+| -------|-------|-------- |
+| yes    | yes   | yes     |
+
+
+Assert that the wrapper contains a exact amount of descendants matching the given selector.
+Defaults to looking for exactly 1 descendent
+
+```js
+import React from 'react'
+import {mount, render, shallow} from 'enzyme'
+
+class Fixture extends React.Component {
+  render () {
+    return (
+      <div id='root'>
+        <span id='child'>
+          <span id='last'></span>
+          <span id='last'></span>
+        </span>
+      </div>
+    )
+  }
+}
+
+const wrapper = mount(<Fixture />) // mount/render/shallow when applicable
+
+expect(wrapper).to.have.descendantsCount('#root')
```

Comment:
this feels like a pattern we would need to redo for every kind of matcher

afair we could do something like a flag 
```
expect(wrapper).to.have.exactly(5).descendants('#root')
```

could become standard pattern => imo less api => less stuff people need to relearn or doublecheck

@vesln might want to chip in - he knows more about this stuff - his call